SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<MasterDetailPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="SevenSlots.MainPage"
+             Title="I am the master"
+             MasterBehavior="Popover">
+
+    <MasterDetailPage.Master>
+        <ContentPage Padding="10" BackgroundColor="Gray" Title="Master">
+            <ContentPage.Content>
+                <StackLayout Margin="5, 30, 5, 5">
+                    <Button Text="Menu Item 1" 
+                            BackgroundColor="MediumVioletRed" 
+                            Clicked="Button_Clicked"
+                            CornerRadius="20"/>
+                    <Button Text="Menu Item 2" 
+                            BackgroundColor="MediumVioletRed"
+                            Clicked="Button_Clicked_1"
+                            CornerRadius="20"/>
+
```

Comment:
Isn't there some sort of general style so we do not have duplicated code?

Reply:
Haven't searched that yet, but will do in a future possible task.

Reply:
Done! Buttons have now a common global style!